src/test/java/org/abstractj/kalium/crypto/StreamTest.java

```diff
@@ -0,0 +1,33 @@
+package org.abstractj.kalium.crypto;
+
+import org.abstractj.kalium.NaCl;
+import org.junit.BeforeClass;
+import org.junit.Test;
+

```

Comment:
I proposed TestVectors.java modification twice. The last one is correct, please ignore the first one